src/main/resources/com/amazon/jenkins/ec2fleet/EC2FleetCloud/config.jelly

```diff
@@ -31,7 +31,7 @@
 
     <f:description>Fleet list will be available once region and credentials are specified. Only maintain supported, see help</f:description>
     <f:entry title="${%EC2 Fleet}" field="fleet">
-        <f:select/>
+        <f:select checkMethod="post"/>
```

Comment:
is this required? Does it make any difference to the validation? 🤔

Reply:
Previously was trying a different implementation, and was getting an error `[Jenkins: 403 No valid crumb was included in the request](https://stackoverflow.com/questions/44711696/jenkins-403-no-valid-crumb-was-included-in-the-request)`, which required making the check method a post request. But, should not need it for the current approach. Thanks, updating